File: go/comparison_operators.go
package main

import "fmt"

func main() {
	var result = 1 == 2
	fmt.Printf("1 equals 2 is %v\n", result)
	
	result = 1 != 2
	fmt.Printf("1 not equals 2  is %v\n", result)
	
	result = 1 > 2
	fmt.Printf("1 larger than 2 is %v\n", result)
	
	result = 1 >= 2
	fmt.Printf("1 larger than or equals 2 is %v\n", result)
	
	result = 1 < 2
	fmt.Printf("1 less than 2 is %v\n", result)
	
	result = 1 <= 2
	fmt.Printf("1 less than or equals 2 is %v", result)
}

File: src/c/comparison_operators.c
#include <stdio.h>

int main()
{
    int result = 1 == 2;
    printf("1 equals 2 is %d\n", result);
	
    result = 1 != 2;
    printf("1 not equals 2 is %d\n", result);
	
	result = 1 > 2;
	printf("1 larger than 2 is %d\n", result);
	
	result = 1 >= 2;
	printf("1 larger than or equals 2 is %d\n", result);
	
	result = 1 < 2;
	printf("1 less than 2 is %d\n", result);
	
	result = 1 <= 2;
	printf("1 less than or equals 2 is %d", result);

	return 0;
}